superset/assets/stylesheets/superset.less

```diff
@@ -536,3 +536,7 @@ tr.reactable-column-header th.reactable-header-sortable {
 .align-right {
   text-align: right;
 }
+.disabled {
```

Comment:
Could you change this to `.navbar-brand.disabled` to be more specific and avoid applying this style to other places inadventently?